app/validators/return-requirements/abstraction-period.validator.js

```diff
@@ -0,0 +1,105 @@
+'use strict'
+
+/**
+ * Validates data submitted for the `/return-requirements/{sessionId}/abstraction-period` page
+ * @module AbstractionPeriodValidator
+ */
+
+const Joi = require('joi').extend(require('@joi/date'))
+
+const { leftPadZeroes } = require('../../presenters/base.presenter.js')
+
+/**
+ * Validates data submitted for the `/return-requirements/{sessionId}/abstraction-period` page
+ *
+ * When setting up a requirement, users must specify an abstraction period for the return requirement. Users must input
+ * a valid from-date and to-date for the licence. If these requirements are not met, the validation will return an
+ * error.
+ *
+ * @param {Object} payload - The payload from the request to be validated.
+ *
+ * @returns {Object} - The result from calling Joi's schema.validate(). If any errors are found the 'error: ' property
+ * will also exist, detailing what the issue is.
+ */
+function go (payload) {
+  const {
+    'fromAbstractionPeriod-day': fromDay,
```

Comment:
Having a mix of[ kebab case](https://www.theserverside.com/definition/Kebab-case) and [Camel case](https://www.techtarget.com/whatis/definition/CamelCase) doesn't look right here. From viewing one of [Fazels PR's ](https://github.com/DEFRA/water-abstraction-system/pull/646) I would say we need to stick to kebab case because of the way it has come from the page. I think this was something that was discussed during a dev call to sort for our coding conventions.